lesson07/assignment/cli3/src/views/Accordion.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div class="home">
```

Comment:
You've got this working static, just need the content into data and loop over it.